upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -107,20 +108,128 @@ backup of Workload Manager configuration data and files is created. Once complet
    /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prepare-assets.sh --csm-version csm-${CSM_RELEASE} --tarball-file "${CSM_TAR_PATH}"
    ```
 
-## Stage 0.2 - Update SLS
+## Stage 0.2 - Plan and coordinate network upgrade
 
 ### Abstract (Stage 0.2)
 
-CSM 1.2 introduces the bifurcated CAN (BICAN) as well as network configuration controlled by data in SLS. An offline upgrade of SLS data is performed. For more details on the
+Prior to CSM 1.2, the single Customer Access Network (CAN) carried both the admin network traffic and the user network
+traffic. CSM 1.2 introduces bifurcated CAN (BICAN), which is designed to separate admin network traffic and user network traffic.
+With BICAN, the pre-1.2 CAN network is split into two separate networks:
+
+1. Customer Management Network (CMN)
+
+   This network allows only system administrative access from the customer site. The pre-1.2 CAN is renamed to CMN. By
+   the end of the CSM 1.2 upgrade, all non-administrative access, such as from UANs, will be removed from CMN.
+
+   During the CSM 1.2 upgrade, UANs will retain their pre-1.2 CAN IPs in order to minimize disruption to UANs. However,
+   toward the end of the CSM 1.2 upgrade, UANs will stop registering themselves on CMN and will receive new IPs on the
+   CAN/CHN network below. This process is described in more detail in [UAN Migration](#uan-migration).
```

Comment:
`bican_enable.md` has this. Additionally [CASMINST-4798](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4798) will cover updates to bican_enable and process overview of UAN cutover.

Reply:
@trad511 I can reference `bican_enable.md` as the first thing in in the UAN Migration section below. Please let me know if you want to keep this overview here altogether or just let the user go to the `bican_enable.md` docs -- imo a general overview here helps because the bican_enable.md looks like it needs to be followed after the SLS update, which is a section after UAN Migration warnings and remediation.

Reply:
I referenced `bican_enable.md#minimize-uan-downtime` in lines 217 and 222 below. Feel free to reopen this comment if you think something else needs to be done.